### ISSUE TITLE
Installing Shared Resources 0.9.0 breaks eXide 2.4.9 [BUG]

### ISSUE THREAD

**Issue by raydleemsc** [eXist-db/exist#3524]
**Describe the bug**
Default installation of eXist 5.2.0 includes eXide 2.4.9 and Shared Resources 0.8.4, and updating Shared Resources to 0.8.5 is ok, but updating Shared Resources to 0.9.0 stops eXide from responding and refreshing eXide just gets this - https://gyazo.com/8d639d8dbbf20fd3c98249dc9bf61f5e

**Expected behavior**
eXide blank file

**To Reproduce**
Install eXist 5.2.0 using defaults
Update Shared Resources to 0.9.0

**Context (please always complete the following information):**
 - OS: Windows 10
 - eXist-db version: 5.2.0
 - Java Version Java1.8.0_261

**Additional context**
- How is eXist-db installed? jar
- Any custom changes in e.g. `conf.xml`? No

**Comment by joewiz** [comment]
Does performing the steps in the eXide dialog from your screenshot have any effect? Are there errors in the developer console of your browser? Are there any errors in exist.log?

**Comment by wolfgangmm** [comment]
@joewiz Looks like shared-resources 0.9.0 is missing the ace library required by eXide:

![image](https://user-images.githubusercontent.com/607433/93022262-a94f8480-f5e8-11ea-8f0f-468fe91a99d7.png)

**Comment by raydleemsc** [comment]
> @joewiz Looks like shared-resources 0.9.0 is missing the ace library required by eXide:
> 
> ![image](https://user-images.githubusercontent.com/607433/93022262-a94f8480-f5e8-11ea-8f0f-468fe91a99d7.png)

Excellent, any ideas when it'll get included?

**Comment by joewiz** [comment]
@raydleemsc @wolfgangmm I’ve pulled shared-resources v0.9 from the public repo until I can determine why the resources in the ace submodule weren’t included in the archive. I will publish v0.9.1 with the fix as soon as I have it.

**Comment by joewiz** [comment]
@raydleemsc @wolfgangmm Thanks for the report and the hint about the cause. There must have been some problem in the way the submodule was initialized on my system. My apologies. A clean clone succeeded for me today though, and I confirmed the repackaged version works with eXide on my system, so I've just published it as v0.9.1:

- https://github.com/eXist-db/shared-resources/releases/tag/v0.9.1
- https://exist-db.org/exist/apps/public-repo/packages/shared.html?eXist-db-min-version=3.0.4
